packages/tradershub/src/features/cfd/modals/DxtradePasswordModal/DxtradePasswordFooter.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useDxtradeAccountsList } from '@deriv/api';
+import useDxtradeAccountHandler from '../../../../hooks/useDxtradeAccountHandler';
+import { PlatformDetails } from '../../constants';
+import AddAccountButtonsGroup from '../ButtonGroups/AddAccountButtonsGroup';
+import SuccessButtonGroup from '../ButtonGroups/SuccessButtonGroup';
+import DxtradeCreateAccountButton from './DxtradeCreateAccountButton';
+
+type TDxtradePasswordFooterProps = {
+    password: string;
+};
+
+const DxtradePasswordFooter = ({ password }: TDxtradePasswordFooterProps) => {
+    const { data: dxtradeAccounts } = useDxtradeAccountsList();
+    const { createOtherCFDAccountSuccess } = useDxtradeAccountHandler();
+    const hasDxtradeAccount = dxtradeAccounts?.find(account => account.login);
+
+    if (createOtherCFDAccountSuccess) return <SuccessButtonGroup />;
+
+    if (hasDxtradeAccount) return <AddAccountButtonsGroup password={password} />;
+
+    return (
+        <DxtradeCreateAccountButton
+            buttonText={`Create ${PlatformDetails.dxtrade.title} password`}
```

Comment:
Do we need to pass buttonText if the component is already dxtrade create account button ?
i mean like.....

Reply:
I dont think as the name suggests its now a specific entity based one so we need to move it into component because it is not a generic component anymore.
@shontzu-deriv

Reply:
in some places the button text needs to be "add account"

```JavaScript
 <DxtradeCreateAccountButton buttonText='Add account' password={password} />
```

Reply:
Then it makes sense

Reply:
cool

Reply:
Osem